config/gregtech/worldgen/overworld/lignitecoal_vein.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
I'm pretty sure you have another file for this already, but double check.

Reply:
i dont